### PR TITLE
Updated JSConfs globally

### PR DESCRIPTION
**Pull request by Stephn-R**
- updated links as well

### CODE REVIEW

**Comment by Stephn-R** [comment]
@RichardLitt

File: README.md [hunk 1]
```diff
@@ -39,8 +39,8 @@ _Many of the links above are to Model View Culture's issue on events. I have lin
 * [HTML5DevConf](http://html5devconf.com/) // [@html5devconf](https://twitter.com/html5devconf)
 * [JavaOne](https://www.oracle.com/javaone) // [@JavaOneConf](https://twitter.com/JavaOneConf)
 * [jQuery Conference](http://events.jquery.org/2014/chicago/) // [@jqcon](https://twitter.com/jqcon)
-* [JSConf US](http://jsconf.us) // [@JSConfUS](https://twitter.com/JSConfUS)
-* [JSConf](http://jsconf.com) // [@jsconf](https://twitter.com/jsconf)
+* [JSConf (All)](http://jsconf.com) // [@jsconf](https://twitter.com/jsconf)
```
Comment (RichardLitt) [inline README.md]: All should probably be in Misc, now. We need a better "global" setting.

File: README.md [hunk 2]
```diff
@@ -64,7 +64,7 @@ _Many of the links above are to Model View Culture's issue on events. I have lin
 * [TXJS](http://texasjavascript.com) // [@txjs](https://twitter.com/txjs)
 * [XOXO](http://2014.xoxofest.com/) // [@xoxox](https://twitter.com/xoxo)
 
-####New York
+###New York
```
Comment (RichardLitt) [inline README.md]: This should be four - it's part of North America. I only really have it because I was living there. Can you change it back to 4 hashtags?

File: README.md [hunk 3]
```diff
@@ -86,7 +86,7 @@ _Many of the links above are to Model View Culture's issue on events. I have lin
 * [Front Interior](http://www.frontinterior.com.br/) // [@frontinterior](https://twitter.com/frontinterior)
 * [JSConf Argentina](http://www.jsconfar.com/) // [@jsconfar](https://twitter.com/jsconfar)
 * [JSConf BR](http://jsconfbr.org) // [@jsconfbr](https://twitter.com/jsconfbr)
-* [JSConf Colombia](http://jsconf.co/) // [@jsconfco](https://twitter.com/jsconfco)
+* [JSConf Co](http://jsconf.co/) // [@jsconfco](https://twitter.com/jsconfco)
```
Comment (RichardLitt) [inline README.md]: Why the change? Colombia is more expressive, especially in a list.

**Comment by RichardLitt** [comment]
Thanks man! Added a couple of notes, before merging. :)

File: README.md (hunk 1 above)
Comment (Stephn-R) [inline README.md]: What about a section labeled "Organizations"

Comment (RichardLitt) [inline README.md]: That sounds pretty good to me. Since JSConf on it's own isn't a thing.
